services/src/main/java/org/keycloak/services/managers/RealmManagerProvider.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.services.managers;
+
+import org.keycloak.provider.Provider;
+
+/**
+ * This provider has been created to listen to {@link org.keycloak.storage.ImportRealmFromRepresentation} events.
+ * If that is no longer needed after further steps around the legacy storage migration, it can be removed.
+ *
+ * @author Alexander Schwartz
+ */
+public class RealmManagerProvider implements Provider {
```

Comment:
Please merge with `RealmManagerProviderFactory`.

Reply:
@hmlnarik - merged `RealmManagerProvider` into `RealmManagerProviderFactory`, so there is no longer a class named `RealmManagerProvider`. 

Local test successful, builds are still running. Will re-request a review once the build is complete.